test/e2e/network/dns.go

```diff
@@ -88,6 +90,38 @@ var _ = common.SIGDescribe("DNS", func() {
 		validateDNSResults(ctx, f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
+	/*
+		Release: v1.30
+		Testname: DNS, cluster
+		Description: When a Pod is created, the container hostname should match with spec hostname after setting the hostNetwork to true
+	*/
+	framework.ConformanceIt("should resolve the hostname and hostNetwork for a Pod", func(ctx context.Context) {
```

Comment:
```suggestion
	  ginkgo.It("should resolve the hostname and hostNetwork for a Pod", func(ctx context.Context) {
```

Conformance has more strict requirements, you can not promote a test directly to conformance, just add it as any other normal test

Reply:
Done